scripts/clone_data.py

```diff
@@ -0,0 +1,70 @@
+"""Script to clone remote Samudra data locally."""
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "xarray[io]",
+#     "dask",
+# ]
+# ///
+
+import argparse
+import pathlib
+
+import dask
+import dask.diagnostics
+import xarray as xr
+
+DATA_ROOT = "https://nyu1.osn.mghpcc.org/m2lines-pubs/Samudra/"
+
+
+def main(dest_root: pathlib.Path, time_slice: slice) -> None:
+    """Clones slice of Samudra data at the `dest_root` directory."""
+    # Ensure the path/to/dest exists
+    dest_root.mkdir(parents=True, exist_ok=True)
+
+    for name, dest_fmt in [
+        ("OM4", "zarr"),
+        ("OM4_means", "netcdf"),
+        ("OM4_stds", "netcdf"),
+    ]:
+        dest = dest_root / name
+
+        if name == "OM4":
+            data = xr.open_dataset(
+                DATA_ROOT + name, engine="zarr", chunks={"time": 700}
+            )
+            data = data.isel(time=time_slice)
+            data.chunk({"time": 1})
```

Comment:
```suggestion
            data = data.chunk({"time": 1})
```

chunk() is not an in-place operation

Reply:
Based on my tests with it, it seems to slow down saving when we rechunk with time=1

Reply:
That would make sense to me! Though, changing the chunks at rest is a significant data engineering change. I'd rather save stuff that like for the future.